src/index.pug

```diff
@@ -0,0 +1,22 @@
+doctype 
+html
+  head
+    title adding some content for the website
+    link(rel="stylesheet", href = "style.css")
```

Comment:
remove extra spaces from `link(rel="stylesheet", href = "style.css")`
to `link(rel="stylesheet", href="style.css")`